src/pages/Pokemon/index.js

```diff
@@ -1,11 +1,31 @@
 import React from "react";
-import { useParams } from "react-router-dom";
+import { useLocation } from "react-router-dom";
+
+import Heading from "./Heading";
+import Statssmall from "./Statssmall";
+import Nav from "./Nav";
+import Pagination from "./Pagination";
+import Stats from "./Stats";
+import Image from "./Image";
+import Info from "./Info";
 
 const Test = () => {
-  let params = useParams();
+  const { state } = useLocation();
+  const pokemonItem = state?.pokemonItem;
```

Comment:
mate you can PURGE this and then YAALALAALALALALALALALALALALALALALALALALALALALALALALA